trade-finance-manager-api/src/v1/api.js

```diff
@@ -229,6 +293,12 @@ const findFacilitesByDealId = async (dealId) => {
 
 const updateFacility = async (facilityId, facilityUpdate) => {
   try {
+    const isValidFacilityId = isValidMongoId(facilityId);
+
+    if (!isValidFacilityId) {
+      return { status: 400, data: 'Invalid facility id' };
```

Comment:
Will all these returned be caught and consoled error at the calling side or should we add `console.error` for all the non happy branches?